<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="back_img">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-40</y>
     <width>1900</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img_2/imgonline-com-ua-Blur-6Mz5CZJd1C4ptg.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_win">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>740</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(180, 67, 122);
	border: 1px solid rgb(12, 23, 29);
	border-radius:  30px;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(74, 24, 45), stop: 1 rgb(141, 44, 78));
}</string>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="hihi_credit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>561</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img_2/credit).png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>541</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 149, 231);
border: 1px solid rgb(12, 23, 29);
	border-radius:  30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_step">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>475</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rate_num">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
background-color: rgb(186, 71, 128);
	border: 1px solid rgb(14, 25, 31);
}</string>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="am_numb">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
	background-color: rgb(186, 71, 128);
	border: 1px solid rgb(14, 25, 31);
}</string>
   </property>
   <property name="maximum">
    <double>9999999999999999827367757839185598317239782875580932278577147150336.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="amount">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img_2/cred_am.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="term">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img_2/term.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="rate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img_2/rate.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="type">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img_2/type.png</pixmap>
   </property>
  </widget>
  <widget class="QRadioButton" name="differentiated">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>490</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Differentiated</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="annuity">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>490</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Annuity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>200</y>
     <width>891</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 149, 231);
border: 1px solid rgb(12, 23, 29);
	border-radius:  30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>560</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(180, 67, 122);
	border: 1px solid rgb(12, 23, 29);
	border-radius:  30px;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(74, 24, 45), stop: 1 rgb(141, 44, 78));
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>230</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img_2/res.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="month">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>330</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img_2/month.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="over">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>400</y>
     <width>451</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img_2/over.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="total">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>530</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img_2/6ce4c2c47054779d50899fa40ee3e8f2.png</pixmap>
   </property>
  </widget>
  <widget class="QSpinBox" name="term_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
	background-color: rgb(186, 71, 128);
	border: 1px solid rgb(14, 25, 31);
}</string>
   </property>
   <property name="maximum">
    <number>600</number>
   </property>
  </widget>
  <widget class="QLabel" name="over_l">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>430</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="total_l">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>540</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ii">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>290</y>
     <width>441</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 149, 231);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
